VCDMExplainer.md

```diff
@@ -0,0 +1,89 @@
+#Verfiable Credentials Data-Model Explainer
```

Comment:
Need a space after the heading `#` marks throughout the file.

Reply:
Thanks. I'll get these when I add more information.

Reply:
Got the rest, missed this one. ;-)